src/index.ts

```diff
@@ -82,16 +152,16 @@ export default class CodeTool {
   /**
    * Create Tool's view
    *
-   * @returns {HTMLElement}
+   * @returns {HTMLDivElement}
    * @private
    */
-  drawView() {
-    const wrapper = document.createElement('div'),
-        textarea = document.createElement('textarea');
+  private drawView(): HTMLDivElement {
+    const wrapper = document.createElement('div') as HTMLDivElement;
+    const textarea = document.createElement('textarea');
 
     wrapper.classList.add(this.CSS.baseClass, this.CSS.wrapper);
     textarea.classList.add(this.CSS.textarea, this.CSS.input);
-    textarea.textContent = this.data.code;
+    textarea.textContent = this._data.code;
```

Comment:
why it was changed? If you think, that "data" getter is useless, you need to remove it, or use it instead of direct access to `_data`